openapi/openapi_spec.json

```diff
@@ -410,6 +416,115 @@
         ]
       }
     },
+    "/customers/{customer_id}/payment_methods": {
```

Comment:
Both this endpoint and `"/customers/payment_methods" are saying that "list payment methods for a customer, which seems a little confusing. 
Though I would prefer things to be explicit, I'm assuming some need for returning the current customers payment methods exist. So better to clearly label that in the openapi spec/api docs. Clearly mention `/customers/payment_method` returns for which customer(`current?`) and if so how is the customer identified.

Reply:
/customers/payment_method, will return paymentMethod for customer that is tied to its client-secret passed in the params.